services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+    user = get_user_model().objects.create_user(username=username,
+                                                password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)
+
+
+def update_user(user_id: int,
+                username: str = None,
+                password: str = None,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
```

Comment:
Yurii mentioned:
_will user be created correctly with email as None_
and you made None as a default value) This is wrong, f.e. email field cannot be None, fix everywhere

Reply:
But if we don't apply a value to a charfield in django - it'll set blank by default or i'm wrong?

Reply:
I see, you talked about validation on line 36-49, it still can be improved by taking kwargs instead of checking each value 1 by 1, and use for loop

Reply:
Same in create user

Reply:
Oh, now I understand, I`ll fix it soon, thank you!